.github/workflows/check-codestyle-filtered.yml

```diff
@@ -111,6 +111,13 @@ jobs:
             if [[ "$clippy_warnings" -gt "$expected_clippy_warnings" ]]; then
               echo "Clippy reported $clippy_warnings warnings, which is higher than the upstream number of $expected_clippy_warnings warnings."
               echo "Please make sure you've run \`cargo clippy\` on the Rust files you've modified."
+
+              if [[ -f "expected-clippy-warnings-raw" ]]; then
+                clippy_warnings_raw=$(cat clippy-warnings-raw)
+                wget https://github.com/dandavison/delta/releases/download/0.14.0/git-delta_0.14.0_amd64.deb
+                dpkg -i ./git-delta_0.14.0_amd64.deb
+                delta clippy-warnings-raw clippy-warnings --paging never --side-by-side
```

Comment:
Nit: Could this just use `diff`?

Reply:
I felt side-by-side diff is nicer, but probably we can tune it later. I'll use diff for now as suggested.